core/templates/domain/collection/read-only-collection-backend-api.service.ts

```diff
@@ -91,10 +91,14 @@ export class ReadOnlyCollectionBackendApiService {
 
   private _cacheCollectionDetails(
       details: ReadOnlyCollectionBackendResponse): void {
-    this._collectionDetailsCache[details.collection.id] = {
-      canEdit: details.can_edit,
-      title: details.collection.title,
-    };
+    if (details.collection.id && details.collection.title) {
+      this._collectionDetailsCache[details.collection.id] = {
+        canEdit: details.can_edit,
+        title: details.collection.title,
+      };
+    } else {
+      throw new Error('Id and title are possibly null!');
```

Comment:
ditto

Reply:
Will cover this file later, I have removed it for now